docs/environments/testnet.md

```diff
@@ -9,12 +9,24 @@ test new applications and experiment without risk before moving onto mainnet.
 Decred is currently on its 3rd testnet, also known as testnet3. Testnets are
 periodically reset to help keep a manageable blockchain file size.
 
+Special rules apply to testnet in order to make it easier to work with.
+For example, if no blocks are found on testnet in a 10 minute period, the PoW
+difficulty will drop so that a block can be generated using the dcrd `generate`
```

Comment:
The formatting on `generate` is slightly jarring...Also, the example command isn't introduced in text. This is OK, but would suggest something like:

...can be generated using dcrd's generate command, as shown below:

```no-highlight
dcrctl --testnet generate 1
```

Reply:
Implemented this suggestion